.gitignore

```diff
@@ -0,0 +1,9 @@
+#  Local .terraform directories
```

Comment:
I'm not sure that this file is necessary. The module is generally used and tested through other projects, which will have their own `.gitignore` directives.